adcc/backends/test_backends_pcm.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import unittest
+import itertools
+import adcc
+import adcc.backends
+import psi4
```

Comment:
Import in Psi-specific routine, otherwise loading this file fails (see CI) if Psi not available.